schemas/stsci.edu/asdf/0.1.0/asdf-schema.yaml

```diff
@@ -0,0 +1,46 @@
+%YAML 1.1
+---
+$schema: "http://json-schema.org/draft-04/schema"
+id: "http://stsci.edu/schemas/asdf/0.1.0/asdf-schema"
+title:
+  ASDF schema
+description: |
+  A metaschema extending YAML Schema and JSON Schema to add support
+  for some ASDF-specific checks, related to nd-arrays.
```

Comment:
And to just elaborate a little bit more -- this is no different than what YAML schema does to add new validation keywords to JSON schema.  This is just adding some more -- but since these are specific to ASDF (specifically to how ndarrays work) and are not generic to YAML, I added them in a new "layer" on top of YAML schema.  So we have a heirarchy of `ASDF schema -> YAML schema -> JSON schema`.